docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -996,49 +1068,151 @@ The UI design consists of three parts:
 - A place for users to manage their owned objects to be ready for escrow trading called `Manage Objects`.
 - A place for users to discover, create, and execute trades called `Escrows`.
 
-:::warning
+### Scaffold a new app
 
-The following code snippets are not the full source code. The snippets are meant to focus on relevant logic important to the functionality of the example and features of Sui.
+The first step is to set up the client app. Run the following command to scaffold a new app.
 
-:::
+```bash
```

Comment:
We have an `npm2yarn` script, but I think it has to include `npm` as seen here: https://github.com/MystenLabs/sui/blob/1dd888add82f71810f1ef4992092169f19260c09/docs/content/guides/developer/first-app/client-tssdk.mdx?plain=1#L35

Instead, we could use the Tabs component, which doesn't need importing (just use it on the page). It would be something like:
```
<Tabs groupId="packagemanager">
<TabItem label="PNPM" value="pnpm">
`pnpm create @mysten/dapp --template react-client-dapp`
</TabItem>
<TabItem label="Yarn" value="yarn">
`yarn create @mysten/dapp --template react-client-dapp`
</TabItem>
</Tabs>
```